src/context/mod.rs

```diff
@@ -511,6 +511,45 @@ impl Context {
         }
     }
 
+    #[cfg(all(any(
+        feature = "min-redis-compatibility-version-7-2",
+        feature = "min-valkey-compatibility-version-8-0"
```

Comment:
RedisModule_AuthenticateClientWithACLUser is present in Redis7.0.  Can we add `feature = "min-redis-compatibility-version-7-0"`?